spec/models/food_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+require 'shoulda/matchers'
+
+RSpec.describe Food, type: :model do
+  describe 'Food model' do
+    describe 'associations' do
+      it { should belong_to(:user) }
+      it { should have_many(:recipe_foods).dependent(:destroy) }
```

Comment:
- Hi, these lines are making your test fails in three cases. Please, verify to fix it.


![image](https://github.com/0sugo/recipe-app/assets/59397159/7a58cc6f-80d9-4091-8e3f-fffdce8f82a9)

Reply:
Thanks @zieeco .Resolved